src/network/networkd-routing-policy-rule.c

```diff
@@ -476,19 +482,21 @@ static int routing_policy_rule_set_netlink_message(const RoutingPolicyRule *rule
                         return r;
         }
 
-        if (rule->table < 256) {
-                r = sd_rtnl_message_routing_policy_rule_set_table(m, rule->table);
-                if (r < 0)
-                        return r;
-        } else {
+        if (rule->l3mdev) {
```

Comment:
No brackets around single line if blocks, as per https://systemd.io/CODING_STYLE/

Reply:
Does it still apply if there's an else block following?

Reply:
Yes.